docs/authoring-components.md

```diff
@@ -725,6 +725,10 @@ Concretely:
   contains javascript, ensure that its component namespace is exported within
   `material-components-web`, and it is registered with `mdc-auto-init`. Lastly, remember to add it
   to `package.json` of `material-components-web`.
+- Ensure that the correct **commit subject** for the package is added to the
```

Comment:
> Do you mean if this package is added to config.validate-commit-msg.scope.allowed then its commit message will be checked and if I wrote wrong commit message like "feat(mdc-grid-list): Implement mdc-grid-list (#47)" it won't go through?
That's correct!

> What about cases if I forget to add this package to that config file, will that commit message go through?
It won't. So this ensures that either you:

- Use a pre-defined scope
- Add the new component scope to the config